samples/scripts/databricks.fsx

```diff
@@ -0,0 +1,29 @@
+#r @"./libs/Newtonsoft.Json.dll"
+#r @"../../src/Farmer/bin/Debug/netstandard2.0/Farmer.dll"
+
+open Farmer
+open Farmer.Builders
+open Farmer.Databricks
+
+let workspace = databricksWorkspace {
+    name "my-databricks-workspace"
+    sku Databricks.Sku.Standard
+    
+    use_public_ip Enabled
+    
+    key_vault "databricks-kv"
+    encryption_key "workspace-encryption-key" "latest"
+    
+    byov_vnet "databricks-vnet"
+    byov_public_subnet "databricks-pub-snet"
```

Comment:
Design question: Would you ever have public subnet and a private subnet at the same time? Is this permitted (if we're not sure, let's leave it as is).

Reply:
Having both a public and private subnet is required for using Bring Your Own VNet with Databricks. Only having one is not supported as the subnets are used for deploying Network Interfaces into and are attached to the machines in the clusters. I'm not sure if the public subnet is needed if enableNoPublicIp is set to true.